test/src/test/java/org/corfudb/integration/ClusterReconfigIT.java

```diff
@@ -466,8 +467,6 @@ public void killAndHealNode() throws Exception {
                 .open();
         final String data = createStringOfSize(1_000);
         Random r = getRandomNumberGenerator();
-        final AtomicBoolean moreDataToBeWritten = new AtomicBoolean(true);
```

Comment:
Why did you remove this?

Reply:
This is creating a daemon writer thread initially introduced to check if we can write after killing or adding. I anyways have a write after these 2 events (in the main test thread) and fail if these writes fail.
So I feel this extra thread is not required now.